src/app/personside/infotabs/saksoversikt/saksdokumenter/DokumentListeElement.tsx

```diff
@@ -214,22 +218,24 @@ class DokumentListeElement extends React.Component<Props> {
                 }
                 ref={this.dokumentRef}
                 valgt={this.props.dokument === this.props.valgtDokument && this.props.visDokument}
-                klikkbar={kanVises}
+                klikkbar={tilgangTilHoveddokument}
             >
-                <IkonWrapper>{dokumentIkon(kanVises)}</IkonWrapper>
+                <IkonWrapper>{dokumentIkon(this.harTilgangTilJournalpost(dokumentMetadata))}</IkonWrapper>
```

Comment:
dette var kanskje litt lite lesbart..?

Reply:
`getDokumentIkon`..? og i metoden kanskje rename til `<DokumentIkon>` og `<DokumentIkkeTilgangIkon>`..?

Reply:
og kanskje rename metoden `harTilgangTilJournalpost` til `kan åpne dokument` eller noe?

Reply:
Enig i de to første, men journalpost og dokument er forskjellige ting her. Eller kanskje jeg misforstår hva du mener?

Reply:
ikke helt sikker på hva jeg mente 😝